src/translation/translation_es_ES.ts

```diff
@@ -39,12 +39,12 @@
     <message>
         <location filename="../util.cpp" line="597"/>
         <source>(build)</source>
-        <translation type="unfinished"></translation>
+        <translation>(versión)</translation>
```

Comment:
OK, I think we need to add a comment providing context for this and `(runtime)`.
```
 *** Qt cross-platform application framework 6.7.2 (build), 6.7.2 (runtime)
```
`(build)` refers to the framework used at the time Jamulus was built from source.  So the word "build" should come from "build from source" (Google says "construir desde la fuente", so an abstract noun from "construir").
`(runtime)` refers to the framework provided by the platform Jamulus is running on.  So "runtime" should come from "run-time platform".  "tiempo de ejecución" agrees with Google but still doesn't look quite right...  I'm wondering if there's a more common term.

Reply:
Yes, these terms were a bit tricky to translate without context. I think "compile" would make life easier for translators. As for "runtime", I think it's one of those words that is probably used as-is, in Spanish at least. "tiempo de ejecución" turned up in several places as the correct translation but as you say, it sounds awkward. I just don't think there's a widely-used Spanish equivalent.

Reply:
Exactly the same issue in German.

Reply:
@gilgongo do you want to weigh in on this?  Would it violate the style guidance to go with the more technically accurate terms, for clarity?

Reply:
What's the confusion we want to avoid, exactly? Is it that there might be different versions for build and runtime so that you don't want to accidentally specify the wrong one? Sometimes things can benefit from "self-naming" too don't forget. So we just don't explicitly name one of them:

`Qt cross-platform application framework 6.7.2, 6.7.2 (runtime)`

Or am I misunderstanding? :-)

Reply:
You're misunderstanding.

The build chain runs using a specific version of Qt.

Depending on the runtime platform, a completely different version of Qt might be in use, earlier or later.  (And different people on the same platform could have different versions installed.)

The message logs both for diagnostic purposes.

What's currently labelled in English as "build" is the version of Qt used in compiling Jamulus itself from source to executable.

What's currently labelled in English as "runtime" is the version of Qt being used for executing Jamulus on the computer it's running on.

The word "build" in isolation isn't particularly helpful towards translators in capturing the meaning.  I think "compile" would be better (as it's more clearly to do with the technical operation of creating Jamulus, which is what it's referring to).

The word "runtime" isn't specific.  It can mean "how long something has been running" - which is completely wrong here.  We're interested in the version of Qt on the computer Jamulus is executing on.  Or installed on.  So "installation" or "execution" might be more useful.

Reply:
OK I see. I'm not close enough to the English nuances of the different terms on offer, but on your comment about confusion, Ubuntu (usually) names package(s) as:

`<foo> <VersionNumber>-<DebianRevisionNumber>`

eg:

`apache2     2.4.52-1ubuntu4.12 ` 

And assumes people know that 2.4.52 is the "base" Apache version while 4.12 is the Debian one, and not the other way around. That's why I was thinking we could do similar with, say:

`Qt cross-platform application framework 5.15.2 (6.7.2)`

Reply:
@softins can you explain this better?

Reply:
> @softins can you explain this better?

Well I thought you did fine in your comment above. We just want to identify and distinguish the versions of Qt used at build/compile time and at run time. These are only likely to be different on systems such as Linux where the application uses the shared Qt libraries installed by the distro it's running on. On Windows and Mac, the Qt runtime is bundled with the app. Probably also on Android and iOS.

This terminology was introduced in my PR #3231, and there was some discussion in https://github.com/jamulussoftware/jamulus/pull/3231#discussion_r1485219727 about the English terms to be used. I don't think we gave any thought at the time to translation.

Maybe that change needs to be reworked so that the build-time and run-time versions are only shown when they differ?

Reply:
Open an issue please. I don't think this should block merging this PR

Reply:
Or maybe even start with a discussion.  I'm not sure how to reproduce the problem this was addressing now...

Reply:
Ok. Could you please open a discussion and ping all people involved here? I'd really like to get this PR in soon to tag beta2

Reply:
https://github.com/orgs/jamulussoftware/discussions/3356 opened.